src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Interface/IAltinnApp.cs

```diff
@@ -124,8 +124,9 @@ public interface IAltinnApp
         /// <param name="layoutSetId">The layout set id</param>
         /// <param name="currentPage">The current page of the instance.</param>
         /// <param name="dataTypeId">The data type id of the current layout.</param>
+        /// <param name="formData">The form data in json string.</param>
         /// <returns> The pages in sorted order.</returns>
-        virtual async Task<List<string>> GetPageOrder(string org, string app, int instanceOwnerId, Guid instanceGuid, string layoutSetId, string currentPage, string dataTypeId)
+        virtual async Task<List<string>> GetPageOrder(string org, string app, int instanceOwnerId, Guid instanceGuid, string layoutSetId, string currentPage, string dataTypeId, string formData)
```

Comment:
Is there a reason we are using a string for the form data here and not an object? I see that for `Calculate` in the calculation handler we use an object. Should we try to be consistent with this typing? (To make it easier and more familiar for the app developer?)

Reply:
We cannot treat the object the same way as we do with the data.. Simply casting it to (Skjema) throws an exception as it is a json object we receive. 
Figured we shouldn't treat it as an object if it cannot be handled the same way as we usually handle form data that is of the type object.

Reply:
Why can't you treat is as an `object` though? I understand we can't cast it to the specific datamodel object (as this varies from app to app)

Reply:
We could, but isn't it confusing for the app developer if you get the form data as an object, but it can't be treated the same way as form data in other parts of the app?

Reply:
My point is that that what the app developer is used to. Both in `Calculate` and `FormatPdf` for instance the form data is passed as an object. String is the exception :)